tests/openapi/test_strictmode.py

```diff
@@ -852,4 +851,81 @@ def test_strict_mode_write_rate_limiting(collection_name):
             }
         )
 
-        assert response.ok, "Rate limiting should be disabled now"
\ No newline at end of file
+        assert response.ok, "Rate limiting should be disabled now"
+
```

Comment:
Can you please add a basic test with a nested filter for sanity check

Reply:
Done: https://github.com/qdrant/qdrant/pull/5754/files#diff-70aaa118e65b662af3397f4a9fb93751d947b331c0e9bc441dfaf49531dfbc09R934